### ISSUE TITLE
[Bug Report] (Accessibility) v-select component missing "aria-expanded" attribute

### ISSUE THREAD

**Issue by grantmooney** [vuetifyjs/vuetify#6147]
### Versions and Environment
**Vuetify:** 1.4.1
**Vue:** 2.5.22
**Browsers:** Chrome 70.0.3538.102
**OS:** Windows 10

### Steps to reproduce
Mount an instance of the v-select component, and run a WCAG compliance audit (using a tool like axe-core, for example) on the rendered result.

### Expected Behavior
Rendered element with 'role="combobox"' should have required ARIA attribute: "aria-expanded"

### Actual Behavior
Rendered element with 'role="combobox"' does not have "aria-expanded" attribute.

### Reproduction Link
<a href="https://codepen.io/gmooneymcci/pen/yGZXjN" target="_blank">https://codepen.io/gmooneymcci/pen/yGZXjN</a>

### Other comments
This does not conform to WCAG 2 A requirements

**Comment by KaelWD** [comment]
Duplicate of #3999
